---
layout: post
title:  "NIAB_CSSL Research Paper and Data"
date:   2023-02-01 11:11:05 +0000
categories: jekyll update
---


A research paper authored by memebers of the DFW NIAB WP3 team has been published in the The Plant Genome. The article is titled: *A wheat chromosome segment substitution line series supports characterization and use of progenitor genetic variation*.

## Abstract

Genome-wide introgression and substitution lines have been developed in many plant species, enhancing mapping precision, gene discovery, and the identification and exploitation of variation from wild relatives. Created over multiple generations of crossing and/or backcrossing accompanied by marker-assisted selection, the resulting introgression lines are a fixed genetic resource. In this study we report the development of spring wheat (*Triticum aestivum L.*) chromosome segment substitution lines (CSSLs) generated to systematically capture genetic variation from tetraploid (*T. turgidum* ssp. *dicoccoides*) and diploid (*Aegilops tauschii*) progenitor species. Generated in a common genetic background over four generations of backcrossing, this is a base resource for the mapping and characterization of wheat progenitor variation. To facilitate further exploitation the final population was genetically characterized using a high-density genotyping array and a range of agronomic and grain traits assessed to demonstrate the potential use of the populations for trait localization in wheat.
## Link

The paper can be viewed following this [link.](https://doi.org/10.1002/tpg2.20288)

## Data download

Genetic data are available for each set of genotypes and markers used to form the graphical genotypes. Here the marker data are called on: an identical homozygote allele to ‘Paragon’ (‘0’), a heterozygous call (‘1’), an alternative homozygote allele to Paragon (‘2’) or a missing data point (‘NA’). 
-	The NIAB_AB (A genome) CSSLs data are available [here from cerealsdb.](https://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/genotyping_data/NIAB_CSSL_Agenome.csv) 
-	The NIAB_AB (B genome) CSSLs data are available [here from cerealsdb.](https://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/genotyping_data/NIAB_CSSL_Bgenome.csv)
-	The NIAB_D CSSLs data are available [here from cerealsdb.](https://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/genotyping_data/NIAB_CSSL_Dgenome.csv)

Alternatively, genetic marker data are also available for the same genotypes but with the markers ordered on the genetic linkage 35k consensus map positions (available from: www.cerealsdb.uk.net). Here the marker data is called based on the original array calls with some curation and quality control. This version of the data was used to make Supplementary Figures S3, S4 and S5, in the paper. The data are available in a zip folder [here.](https://github.com/NIAB/niab-dfw-wp3/raw/main/data/NIAB_CSSLs_Selection_genotypes_ordered_35k.zip)


# Citation

Horsnell, R., Leigh, F. J., Wright, T. I. C., Burridge, A. J., Ligeza, A., Przewieslik-Allen, A. M., Howell, P., Uauy, C., Edwards, K. J., & Bentley, A. R. (2023). A wheat chromosome segment substitution line series supports characterization and use of progenitor genetic variation. The Plant Genome, 00, e20288. https://doi.org/10.1002/tpg2.20288